aiida_quantumespresso/workflows/functions/create_kpoints_from_distance.py

```diff
@@ -31,7 +32,7 @@ def create_kpoints_from_distance(structure, distance, force_parity):
 
     # If the vectors of the cell all have the same length, the kpoint mesh should be isotropic as well
     if is_symmetric_cell and not is_symmetric_mesh:
-        nk = max(lengths_kpoint)
-        kpoints.set_kpoints_mesh([nk, nk, nk])
+        nkpoints = max(lengths_kpoint)
+        kpoints.set_kpoints_mesh([nkpoints, nkpoints, nkpoints])
```

Comment:
No big deal, but I'm not a fan of letting linters dictate this kind of changes.

Reply:
Thanks for the review Marco. I agree that we shouldn't blindly accept all linter warnings, but I felt in this case it was more clear and therefore justified. There are quite a few warnings that I consciously ignored